Client/Frontend/Home/ReaderPanel.swift

```diff
@@ -378,12 +382,15 @@ class ReadingListPanel: UITableViewController, HomePanel, SWTableViewCellDelegat
 
         containerView.snp_makeConstraints({ (make) -> Void in
             // Let the container wrap around the content
-            make.top.equalTo(overlayView.snp_top).offset(20)
-            make.bottom.equalTo(readingListLabel.snp_bottom)
+            make.top.equalTo(logoImageView.snp_top)
             make.left.equalTo(welcomeLabel).offset(ReadingListPanelUX.WelcomeScreenItemOffset)
             make.right.equalTo(welcomeLabel).offset(ReadingListPanelUX.WelcomeScreenCircleOffset)
+
             // And then center it in the overlay view that sits on top of the UITableView
             make.centerX.equalTo(overlayView)
+            if DeviceInfo.deviceModel().rangeOfString("iPad") != nil {
+                make.centerY.lessThanOrEqualTo(overlayView).offset(-180)
+            }
```

Comment:
If possible, we should avoid creating constraints based on the device. Instead, try to create them using `lessThanOrEqualTo`, priorities, etc. like you did above.